docs/topics/Gateway.md

```diff
@@ -343,7 +343,11 @@ Note that `num_shards` does not relate to, or limit, the total number of potenti
 
 ## Sharding for Very Large Bots
 
-If you own a bot that is in over 250,000 guilds, there are some additional considerations you must take around sharding. Please file a support-ticket to get moved to the sharding for big bots, when you reach this amount of servers. You can contact the discord support using [https://dis.gd/contact](https://dis.gd/contact).
+If you own a bot that is in over 250,000 guilds, there are some additional considerations you must take around sharding. Because of our rate limits of 1000 `IDENTIFY` calls per day, and our limits on how many shards you can start concurrently, you might find yourself unable to fully restart your bot due to the number of shards you are running. Since reconnects are a normal part of our gateway, you may not have enough `IDENTIFY` left to roll out new code, for example.
```

Comment:
Huh, I thought the implication was that very large bots got an increased number of `IDENTIFY` calls. Is that not the case?

Reply:
That is the case. Large sharding bumps from 1k --> 2k. It also (unless I am misinformed) allows you to start more shards concurrently (which is why as part of the process we sync you to a multiple of 16 or 32 when you get _real_ big).

Reply:
So, just to make sure I'm reading your response right... it becomes 2000 `IDENTIFY` calls when you become a Very Large Bot? If so, this should be edited to mention that -- because it seems to say that you still only have 1000 `IDENTIFY` calls.

Reply:
You should also probably change the wording on the paragraph just below to mention the increased limit on starting shards concurrently.

Reply:
I think this might be relevant to the convo (2 paragraphs lower):
> This is not true for _all_ bots above 250,000 guilds, but if you have a large bot and find yourself having these issues, please file a support-ticket to get moved to Large Bot Sharding. You can contact the discord support using [https://dis.gd/contact](https://dis.gd/contact).

So it seems 250k != Large Bot Sharding, and therefore you don't automatically get 2k `IDENTIFY`s when you have a big bot.

Reply:
I thought you should contact them when you are getting random 4000s from the gateway?

Reply:
@A5rocks I understood that, and I wasn't trying to say that 250k means it's a Very Large Bot.

However, when a bot gets Large Bot Sharding enabled, you then get an increase in `IDENTIFY` calls, as well as the ability to start more shard concurrently. As it stands, the documentation does not mention either of these points (and, in fact, seems to imply the opposite on both) -- so I was suggesting that msciotti document both points.

Reply:
Good point, adding both.